src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Category/Category.jsx

```diff
@@ -3,62 +3,37 @@ import * as PropTypes from 'prop-types';
 import {FieldPropTypes} from '~/EditPanel/FormDefinitions/FormData.proptypes';
 import {FastField} from 'formik';
 import {DropdownTreeSelect} from '~/DesignSystem/DropdownTreeSelect';
+import {useQuery} from 'react-apollo-hooks';
+import {GetCategories} from './category.gql-queries';
+import {ProgressOverlay} from '@jahia/react-material';
+import {translate} from 'react-i18next';
+import {adaptToCategoryTree} from './catergory.adapter';
 
-const data = [
-    {
-        label: 'VP Accounting',
-        checked: true,
-        children: [
-            {
-                label: 'iWay',
-                children: [
-                    {label: 'Universidad de Especialidades del Espíritu Santo'},
-                    {label: 'Marmara University'},
-                    {label: 'Baghdad College of Pharmacy'}
-                ]
-            },
-            {
-                label: 'KDB',
-                children: [
-                    {label: 'Latvian University of Agriculture'},
-                    {label: 'Dublin Institute of Technology'}
-                ]
-            },
-            {
-                label: 'Justice',
-                children: [
-                    {label: 'Baylor University'},
-                    {label: 'Massachusetts College of Art'},
-                    {label: 'Universidad Técnica Latinoamericana'},
-                    {label: 'Saint Louis College'},
-                    {label: 'Scott Christian University'}
-                ]
-            },
-            {
-                label: 'Utilization Review',
-                children: [
-                    {label: 'University of Minnesota - Twin Cities Campus'},
-                    {label: 'Moldova State Agricultural University'},
-                    {label: 'Andrews University'},
-                    {label: 'Usmanu Danfodiyo University Sokoto'}
-                ]
-            },
-            {
-                label: 'Norton Utilities',
-                children: [
-                    {label: 'Universidad Autónoma del Caribe'},
-                    {label: 'National University of Uzbekistan'},
-                    {label: 'Ladoke Akintola University of Technology'},
-                    {label: 'Kohat University of Science and Technology  (KUST)'},
-                    {label: 'Hvanneyri Agricultural University'}
-                ]
-            }
-        ]
+// eslint-disable-next-line
```

Comment:
what's the issue with eslint?

Reply:
I'm still not use the field component. I will remove it in the next PR :)